0_RootFS/gcc_common.jl

```diff
@@ -224,15 +224,16 @@ function gcc_script(compiler_target::Platform)
         # Patch for building binutils 2.30+ against FreeBSD
         atomic_patch -p1 $WORKSPACE/srcdir/patches/binutils_freebsd_symbol_versioning.patch || true
 
+        #gprofng doesn't build on musl anymore ;(
         ./configure --prefix=${prefix} \
             --target=${COMPILER_TARGET} \
             --host=${MACHTYPE} \
             --with-sysroot="${sysroot}" \
             --enable-multilib \
             --program-prefix="${COMPILER_TARGET}-" \
             --disable-werror \
-            --enable-deterministic-archives
-
+            --enable-deterministic-archives \
+            --disable-gprofng
```

Comment:
Did you see if in https://git.alpinelinux.org/aports/tree/main/gcc?h=master there's anything relevant?

Reply:
I didn't find anything. https://bugs.gentoo.org/865113 was my reference